tests/py/web_components_test.py

```diff
@@ -0,0 +1,11 @@
+"""Check that our vendored web components are identical between the different themes."""
+
+from pathlib import Path
+
+
+def test_top_nav_bar():
+    furo = Path("qiskit_sphinx_theme/furo/base/static/js/web-components/top-nav-bar.js").read_text()
```

Comment:
Minor, but should it be `read_text(encoding="utf-8")`?

Reply:
Hm yeah, I think technically it matters on Windows. macOS and Linux use utf-8 by default though.